code-challenges.js

```diff
@@ -9,67 +9,185 @@
+    return `${banana} is at boiling point`;
+  } else{
+    return "oopsie....you seem to have tripped on a banana!";
+  }
+}
+
+console.log(compareBoilPoint(temp1))
+console.log(compareBoilPoint(temp2))
+console.log(compareBoilPoint(temp3))
+
 
 
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
+//Write a custom function that takes an array as an arguement
+//write a for loop that starts at index 0 and runs for the length of the array.
+//use .push to send modified array (array times 5) to new let variable
+//Return new arrayXFive variable
+
 var myForNumbers1 = [3, 7, 0, 6, -9]
 var myForNumbers2 = [2, 5, -8, 20, 32]
 
+const multByFive = (array) => {
+  let arrayXFive = []
+  for (let i = 0; i < array.length; i++){
+    arrayXFive.push(array[i] * 5)
+  }
+  return arrayXFive
+}
+
+console.log(multByFive(myForNumbers1))
+console.log(multByFive(myForNumbers2))
+//
+//
+//
+// // --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
+
+//Create a custom function that takes an array as an arguement
+//map array with value returned times 5
+//return mapped array
+//log result
 
+var myMapNumbers1 = [87, 6, 28, 16, 2]
+var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+const mapXFive = (array) => {
+  return array.map(value => value * 5)
+}
 
+console.log(mapXFive(myMapNumbers1));
+console.log(mapXFive(myMapNumbers2));
 
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
-var myMapNumbers1 = [87, 6, 28, 16, 2]
-var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+// // --------------------4) Create a function that removes all the vowels from a string.
+// // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 
+//Create a function that takes a string as an arguement
+//break strings down into arrays
+//filter array with if else statement that determines if there are vowels by using includes
+//join new array into a new string
+//log result
 
 
-// --------------------4) Create a function that removes all the vowels from a string.
-// Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
+//Tried multiple options to get to this, I still feel it's a little clunky for having a string typed out for each vowel, uppercase and lowercase, however all of the other functions I tried seemed to come up with errors.
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+const noVowels = (string) => {
+  let stringArray = string.split("")
+  let noVowelArray = stringArray.filter(value =>{
```

Comment:
`.includes()` already returns `true` or `false` so no need to use an if statement
```suggestion
  let noVowelArray = stringArray.filter(value => !['A','a','E','e','I','i','O','o','U','u'].includes(value));
```

Reply:
there is an opportunity to use a built-in so that you don't have to hard-code both the upper and lower case versions of a vowel